miqcli/collections/automation_requests.py

```diff
@@ -79,6 +79,19 @@ def create(self, method, payload, payload_file,):
                     tenant.get_id(input_data['tenant'])
 
             _payload = OSP_FIP_PAYLOAD
+        elif method == SUPPORTED_AUTOMATE_REQUESTS[2]:
+            # release the floating_ip
+            OSP_FIP_PAYLOAD["uri_parts"]["instance"] = "release_floating_ip"
```

Comment:
`OSP_FIP_PAYLOAD` is a constant, therefore it should not be changed. Wouldn't it be better to be used as a template for a new variable and then change the variable?

Reply:
I agree this needs to be changed.